.gitignore

```diff
@@ -55,6 +55,13 @@ react-native-iap-*.tgz
 # Built files
 tsconfig.tsbuildinfo
 
+# Generated files
 *.js.flow
 *.d.ts
-*.js
```

Comment:
This makes it difficult to search inside IapExample folder since vscode ignores all the js.